src/controllers/action-queue/types.ts

```diff
@@ -81,10 +82,12 @@ export type SwapActionOp = {
   nativeAmount: string
   toTokenId?: string
   toWalletId: string
+  displayKey?: ActionOpDisplayKey
```

Comment:
What if the type for `SwapActionOp['displayKey']` was different from the type for `SeqActionOp['displayKey']` and `ParActionOp['displayKey']`? The reason for this is to treat the keys for a particular action op distinctly from the other action ops. This way a display key for the SwapActionOp is specific to the `SwapActionOp`. This can avoid _some of_ the runtime type error problems we could run into if a key is used on an ActionOp that isn't not suppose to be used on.

Here's an example of what I mean:

```ts
export type SwapActionOp = {
  type: 'swap'
  amountFor: 'from' | 'to'
  fromTokenId?: string
  fromWalletId: string
  nativeAmount: string
  toTokenId?: string
  toWalletId: string
  displayKey?: SwapActionOpDisplayKey
}
export type SwapActionOpDisplayKey = 'swap-deposit' | 'swap-deposit-fees'
```

And here's an example for what we can do with `ParActionOp`:

```ts
export type ParActionOp = {
  type: 'par'
  actions: ActionOp[]
  displayKey?: ParActionOpDisplayKey
}
export type ParActionOpDisplayKey = 'create'
```

Making the set of valid keys specific to the ActionOp avoid the category of error of putting say `swap-deposit` key on ` ParActionOp`. Also it make it very clear to the developer about which keys apply to which `ActionOp`; self-documenting code.